shell/src/test/java/org/apache/accumulo/shell/ShellConfigTest.java

```diff
@@ -123,29 +118,21 @@ public void testTokenAndOptionAndPassword() throws IOException {
   @Test
   public void testZooKeeperHostFallBackToSite() throws Exception {
     ClientConfiguration clientConfig = new ClientConfiguration();
-    Map<String,String> data = new HashMap<>();
-    data.put(Property.INSTANCE_ZK_HOST.getKey(), "site_hostname");
-    AccumuloConfiguration conf = new ConfigurationCopy(data);
-    assertEquals("site_hostname", Shell.getZooKeepers(null, clientConfig));
+    assertFalse("Client config contains zk hosts", clientConfig.containsKey(ClientConfiguration.ClientProperty.INSTANCE_ZK_HOST.getKey()));
+    assertEquals("localhost:2181", Shell.getZooKeepers(null, clientConfig));
```

Comment:
It does not seem that way but the last resort of `Shell.getZooKeepers()` is to make a static call to get the site configuration:
`SiteConfiguration.getInstance(ClientContext.convertClientConfig(clientConfig)).get(Property.INSTANCE_ZK_HOST)`